.travis.yml

```diff
@@ -8,6 +8,9 @@ go:
   - 1.9.1
 install:
   - make bootstrap
+before_script:
+  - make deps
+  - make golint
```

Comment:
Both make deps & make golint are timing out the travis run.
Let us remove make deps
& check if make golint can run only for maya source code. It should not run for vendor/ code

Reply:
`make dep` is not timing out ( target  is `dep ensure` which do not enable for verbose logging mode)
if we do `dep ensure -v` it will show the verbose logging output.

Reply:
It's `make golint` the one that is timing out, I added a 10 minutes timeout for that, and in travis it's taking longer than that.

Maybe reducing the linters that are validated in travis and only make it full for local environment?

Reply:
sounds good @chentex -- running full on local is a good idea.